src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-intro.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: External services
+metaDescription: "This is an introduction to the external services feature"
+redirects:
+  - /docs/apm/apm-ui-pages/monitoring/external-services-page-view-web-cloud-network-data/
+---
+The external services feature allows you to look closely at the upstream and downstream activity of a single service. Those upstream or downstream external services may be your own services you instrumented, or they may refer to services you simply call in the course of a transaction. 
+
+The external services feature is no longer just for New Relic APM agent users. It has expanded to include OpenTelemetry and now uses the power of distributed tracing to give you insights into service activity. The classic external services feature is still available behind a UI toggle, but we recommend you try out the full power the expanded external services.  
+
+This feature provides a combination of maps and graphs to help you:
+
+* Find sources of latency and errors
+* Assess the impact of latency to upstream callers
+* Review transaction-level performance
+* Find traces that involve the transactions you're interested in
+
+To use the external services feature, you need a New Relic account with an installed APM agent with distributed tracing enabled. If you don't already have an account, you can sign up [here](https://newrelic.com/signup) for a free account.
+
+![Screenshot showing the opening map for external services](./images/intro-example.png "Screenshot showing the opening map for external services")
+<figcaption>The thickness of the lines represents the throughput from your service to the upstream or downstream services. When you select a specific service, you will see the various endpoints making calls between the two services.</figcaption>
+
+## Relationship to classic external services [#classic-external-services]
+
+While you can still reach classic external services by using the UI toggle, the main external services UI is now populated by data from distributed tracing. 
+It still provides transaction data similar to classic external services, but here are some key things you need to know about the expanded external services: 
+
+  * **Dependency:** To use the external services feature, you need to enable distributed tracing on services that make calls to each other. 
+  * **Compatibility:** Distributed tracing is not backwards compatible with cross-application tracing, so if you currently rely on classic external services, please note that you will only have visibility into calls between services using the same protocol. 
```

Comment:
I know it's grammatically weird, but the "cross application tracing" functionality never had a dash in it. Since CAT has been deprecated, your call whether to keep your dash or not! :-)

Reply:
@barbnewrelic , good catch! I was trying to follow the unhyphenated original, but I missed this one!